### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by Honey14**

### CODE REVIEW

**Comment by Honey14** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4171** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4171" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/4171?utm_source=stack-comment).

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 - Migrate `BloodSugarEntrySheet` to use `BaseBottomSheet`
 - Migrate `UpdatePhoneNumberDialog` to `BaseDialog`
 - Bump dagger to v2.43.2
-- Bump gradle to v7.5.1
```
Comment (msasikanth) [inline CHANGELOG.md:10]: This is a different Gradle version. Let's add a new entry for the AGP v7.2.2 update.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
...
+- Bump AGP to v7.2.2
```
Comment (msasikanth) [inline CHANGELOG.md:10]: I meant as a separate line. We have update Gradle and AGP as well. 

Gradle is updated to v7.5.1
AGP is updated to v7.2.2

**Review by msasikanth — CHANGES_REQUESTED** [review]
See inline comments

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,8 @@
 - Migrate `LoggedOutOfDeviceDialog` to `BaseDialog`
 - Bump flipper to v0.157.0
 - Fix app crashing when overdue search screen is opened
+- Bump gradle to v7.5.1
```
Comment (janhavisinghh) [inline CHANGELOG.md:22]: We already have `Bump gradle to v7.5.1` changelog at the top

**Review by janhavisinghh — CHANGES_REQUESTED** [review]

**Review by msasikanth — APPROVED** [review]

**Review by janhavisinghh — APPROVED** [review]

**Pull request merged**